src/index.html

```diff
@@ -8,6 +8,111 @@
     <link rel="stylesheet" href="./style.css">
   </head>
   <body>
-    <h1>Stars</h1>
+    <div class="big_star_container">
+      <div class="star_container stars--0">
+        <div class="star_deactivated">
+
+        </div>
+        <div class="star_deactivated">
+
+        </div>
+        <div class="star_deactivated">
+
+        </div>
+        <div class="star_deactivated">
+
+        </div>
+        <div class="star_deactivated">
+
+        </div>
```

Comment:
```suggestion
        <div class="stars__star"></div>
        <div class="stars__star"></div>
        <div class="stars__star"></div>
        <div class="stars__star"></div>
        <div class="stars__star"></div>
```
Fix all cases
![image](https://user-images.githubusercontent.com/96409563/217355259-1a7c1f12-d643-479e-b5b0-02e15b8d5743.png)